strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/npm/NpmArtifactControllerTestIT.java

```diff
@@ -105,6 +106,7 @@ public void testResolveArtifactViaGroupWithProxy(@Remote(url = REMOTE_URL)
 
     @ExtendWith(RepositoryManagementTestExecutionListener.class)
     @Test
+    @Disabled
```

Comment:
Why has this test been `@Disabled`? Could it not be fixed as part of this pull request? If not, could you please clarify what the issue is and how to fix it? Also, if it can't be fixed as part of ths pull request, could you please raise a follow-up task, (although, I'd generally prefer for it to be fixd as part of this pull request)?

Reply:
#1972